mass_mailing_partner/models/res_partner.py

```diff
@@ -56,7 +56,8 @@ def write(self, vals):
                 mm_vals['name'] = vals['name']
             if vals.get('email'):
                 mm_vals['name'] = vals['email']
```

Comment:
I have detected this as incorrect in https://github.com/OCA/social/pull/211/files#diff-7173efc033fbb9673276a0ff5ad30422R58. Maybe is this the problem?

Reply:
Nope, it's another problem, and as I see you're already fixing it, so nothing else to do here.